src/main/java/org/gradle/profiler/gradle/DefaultGradleBuildConfigurationReader.java

```diff
@@ -50,7 +52,8 @@ private void generateInitScript() throws IOException {
             writer.println(
                 "settingsEvaluated {\n" +
                     "   def detailsFile = new File(new URI('" + buildDetails.toURI() + "'))\n" +
-                    "   detailsFile.text = \"isEnterprisePluginApplied=${it.pluginManager.hasPlugin('com.gradle.enterprise')}\\n\"\n" +
+                    "   detailsFile.text = \"isEnterprisePluginApplied=${it.pluginManager.hasPlugin('com.gradle.enterprise')}\\n\" + \n" +
```

Comment:
🤔 Why is there `\\n` before `"` `}\\n\"` is that a bug from before?

Reply:
Oh yeah, it should be ok. We print out:
```
detailsFile.text = "isEnterprisePluginApplied=${it.pluginManager.hasPlugin('com.gradle.enterprise')}
isDevelocityPluginApplied=${it.pluginManager.hasPlugin('com.gradle.develocity')}
"

Nevermind

Reply:
There is a `\\n` since we are writing a property file that should have one entry per line. So in the details file I want to have
```
isEnterprisePluginApplied=true
isDevelocityPluginApplied=true
```

So in the initscript I need:
```
settingsEvaluated { +
   def detailsFile = new File(new URI('<uri>'))
   detailsFile.text = "isEnterprisePluginApplied=${it.pluginManager.hasPlugin('com.gradle.enterprise')}\n" +
       "isDevelocityPluginApplied=${it.pluginManager.hasPlugin('com.gradle.develocity')}\n"
}
```

And that is why I need to double escape some `\n` values.